examples/pytorch/audio-classification/run_audio_classification.py

```diff
@@ -382,18 +385,32 @@ def compute_metrics(eval_pred):
         tokenizer=feature_extractor,
     )
 
+    # mlflow initial
+    experiment_name ='translateion-{}'.format(model_args.model_name_or_path)
+    current_experiment = mlflow.set_experiment(experiment_name)
+    experiment_id=current_experiment.experiment_id
+    mlflow_runner = mlflow.start_run(run_name=model_args.model_name_or_path, experiment_id=experiment_id)
+
     # Training
     if training_args.do_train:
         checkpoint = None
         if training_args.resume_from_checkpoint is not None:
             checkpoint = training_args.resume_from_checkpoint
         elif last_checkpoint is not None:
             checkpoint = last_checkpoint
-        train_result = trainer.train(resume_from_checkpoint=checkpoint)
-        trainer.save_model()
-        trainer.log_metrics("train", train_result.metrics)
-        trainer.save_metrics("train", train_result.metrics)
-        trainer.save_state()
+        
+        with mlflow_runner:
```

Comment:
Looks like it only does one time log_metric. Please make sure it should record loss/throughput every epoch at least.